packages/arb-token-bridge-ui/src/components/syncers/PendingTransactionsUpdater.tsx

```diff
@@ -1,7 +1,7 @@
 import { useCallback, useEffect } from 'react'
 
 import { TransactionReceipt } from '@ethersproject/providers'
-import { Transaction, txnTypeToLayer } from 'token-bridge-sdk'
+import { Transaction, txnTypeToLayer } from 'src/hooks/useTransactions'
```

Comment:
Let's use a relative path here

Reply:
They were present in 3-4 more files. Updated there as well.